### DISCUSSION TITLE
May 19th 2022 - Webhook and Identified Contacts are Working

### DISCUSSION THREAD

**Discussion by jonathonbyrdziak** [merchantprotocol/theadslab#59] [Announcements]
He all, I just wanted to report that the webhook is working, both the creation and the firing are working. Additionally I have rolled out some updates to the old identified contacts page in matomo.

You'll notice there's a new Known Devices page. This will show you your current match rate for known devices and a list of known devices.

I've updated the identified contacts page to "Visitor Profiles" here you'll see the statistics again for match rate and see a list of the visitor profiles we've identified.

**Comment by abadger13** [comment]
@jonathonbyrdziak Google sheets is still not updating (from export or webhook). Is there a way to get the data to sheets? I can only see the dashboard and I'm not sharing this to clients. Thanks.

**Comment by abadger13** [comment]
@jonathonbyrdziak I had a thought -- when I set up the webhook the headers are limited, like in the training video you did detailing how to set up the webhook. I think the new Known Device page is in the full Google Sheet with the full data tabs, right? How do I connect that google sheet to the webhook? Is there a training on this? Is it as simple as making a copy of the full sheet and connecting the webhook to it? Will that work? Thanks. I really need this to start working.

**Comment by FotorecordPS** [comment]
If i delete my current spreadsheet and create a new one, will that make it work? I have not been provided a solution - other than being told its working (which it is not) - still nothing is being reported to my spreadsheet. Please advise.
